packages/client/components/DiscussionThreadListEmptyState.tsx

```diff
@@ -23,10 +40,23 @@ const DiscussionThreadListEmptyState = (props: Props) => {
   const readOnlyMessage = allowTasks
     ? 'No comments or tasks were added here'
     : 'No comments were added here'
-  const message = allowTasks
-    ? '✍️ Be the first to add a comment or task'
-    : '✍️ Be the first to add a comment'
-  return <Message>{isReadOnly ? readOnlyMessage : message}</Message>
+  const message = `Start the conversation${
+    allowTasks ? ' or add takeaway task cards' : ''
+  } to capture next steps.`
+
+  return (
+    <DiscussionThreadEmptyStateRoot>
+      <EmptyDiscussionContainer>
+        <EmptyDiscussionSVG />
+      </EmptyDiscussionContainer>
+      <Message>{isReadOnly ? readOnlyMessage : message}</Message>
+      {!isReadOnly && (
+        <TangledArrowContainer>
+          <TangledArrowSVG />
+        </TangledArrowContainer>
+      )}
```

Comment:
It felt weird to have this arrow pointing to disabled input so I removed it in this case. Is that acceptable @enriquesanchez?

Reply:
Oh, interesting! In what scenarios would this happen?